README.md

```diff
@@ -41,11 +41,11 @@ $ make build
 $ ./nebula-importer --config /path/to/yaml/config/file
 ```
 
-`--config` is used to pass in the path to the YAML configuration file.
+Of them, the `--config` is used to pass the path of the YAML configuration file.
```

Comment:
I would recommend omitting "Of them" to avoid confusion.

"The `--config` option in the above command is used to pass the path of the YAML configuration file."